build/mbl/build-update-payloads.py

```diff
@@ -114,9 +114,66 @@ def main():
             ".testinfo".format(args.image, multi_component_base_path)
         ),
     ]
-    for command in create_update_payload_commands:
+    for command in create_update_bootloader_payloads_commands:
         bitbake.run_command(command, check=True)
 
+    # Create the payloads for the apps testing
+    apps_base_path = (
+        bitbake.builddir / "mbl-core/tutorials/helloworld/release/ipk/"
+    )
+
+    # Payload of a good single app
+    bitbake.run_command(
+        "create-update-payload -a {0}.ipk -o {1}.swu -t {1}.testinfo".format(
+            apps_base_path / "user-sample-app-package_1.0_any",
+            args.outputdir / "user-sample-app-package_1.0_any",
+        ),
+        check=True,
+    )
+
+    # Payload of good 5 apps
+    good_five_apps_cmd = (
+        "create-update-payload -a {0}.ipk {1}.ipk {2}.ipk {3}.ipk {4}.ipk "
```

Comment:
I suggest we use keyword format args, we have too many items in the format arglist and it makes tracing these indexes a headache

Reply:
Fixed.